src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #00ACDC;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+}
+
+.logo {
+  display: inline-block;
+  text-decoration: none;
+  width: 40px;
+  height: 40px;
+
+}
+
+.box-item {
+  margin-top: 22.5;
+  margin-right: 20;
+  margin-bottom: 23;
```

Comment:
Consider using margin only to one side if you use top so use margin-top everywhere